src/components/tour/_tour.scss

```
@@ -0,0 +1,67 @@
+.euiTour--minWidth-default {
+  min-width: calc(#{$euiSizeL} * 10);
```

You don't need to use `calc` here. It's just a straight multiplication of our size variable
```suggestion
  min-width: $euiSizeL * 10;
```